scenarioo-client/app/manage/generalSettings/generalSettings.controller.js

```diff
@@ -43,6 +44,10 @@ function GeneralSettingsController($scope, BranchesResource, ConfigService, Vers
             }
         );
 
+        DiffViewerConfigResource.get(function onSuccess(result) {
+            vm.diffViewerAvailable = result.graphicsMagickAvailable;
+        });
+
         ConfigService.load();
         loadSearchEngineStatus();
```

Comment:
put the other status loading code together with other status code into this private mehtod here and rename it to `loadScenariooStatus`